src/api-v1.js

```diff
@@ -580,6 +580,94 @@ api.declare({
 
 api.declare({
   method: 'put',
+  route: '/ami-set/:amiSetId',
```

Comment:
let's go with just `:id` here as well.  When inside the managing functions for a thing itself, i find it easier to just refer to it as 'id' or something, and use more descriptive names for the ids of things that are referenced as a part of a thing.